library/core/src/char/methods.rs

```diff
@@ -9,6 +9,21 @@ use crate::unicode::{self, conversions};
 use super::*;
 
 impl char {
+    /// The lowest valid code point a `char` can have, `'\0'`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # fn something_which_returns_char() -> char { 'a' }
+    /// let c: char = something_which_returns_char();
+    /// assert!(char::MIN <= c);
+    ///
+    /// let value_at_min = char::MIN as u32;
```

Comment:
I'd rather this be `u32::from(char::MIN)`. (Probably same for `char::MAX` since I'm guessing you copied the docs from that.)

Reply:
Good point; I've updated both to use `u32::from`.